src/Microsoft.AspNetCore.Server.Testing/Deployers/ApplicationDeployer.cs

```diff
@@ -141,8 +140,8 @@ private void KillProcess(int processId)
             {
                 startInfo = new ProcessStartInfo
                 {
-                    FileName = "pkill",
-                    Arguments = $"-TERM -P {processId}",
+                    FileName = "kill",
+                    Arguments = $"-TERM {processId}",
                 };
```

Comment:
Interesting. I just stole this code from dotnet-watch and the idea was that it would kill process tree. Apparently this is not the case - now when I read I see it only seems to kill child processes and not the process itself. The new command however seems to kill only the process but not the child process which won't work if Deployer creates child processes.